testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -364,9 +362,7 @@ def test_ces_error_estimation(self):
         assert_almost_equal(stdev, expected_stdev, decimal=0,
                             err_msg="Unexpected standard daviation  for bootstrapped samples in Clustering Ensemble similarity")
 
-    @dec.skipif(module_not_found('sklearn'),
-                "Test skipped because sklearn is not available.")
-    @dec.slow
+    @pytest.mark.skipif(module_not_found('sklearn'), reason="Test skipped because sklearn is not available.")
```

Comment:
correct here is [`pytest.importorskip`](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency). Note you need to use it inside of the function.